paddle/go/pserver/lib/client/main.go

```diff
@@ -0,0 +1,239 @@
+package main
```

Comment:
I was thinking this need to be integrated into Paddle C++ code (a lot of pointers are used, not sure how much does CPython supports pointer), maybe I am wrong. I think we will know better when we begin to integrate.